mysite/templates/registration/password_change_done.html

```diff
@@ -0,0 +1,8 @@
+{% extends "ct/portal.html" %}
```

Comment:
@alexbojko our new templates should extend something new but not portal template.